### DISCUSSION TITLE
Opinions on renaming the "Position (U)" column header to just "U" to save space

### DISCUSSION THREAD

**Discussion by bluikko** [netbox-community/netbox#8268] [Ideas]
Currently in the "Devices" list `/dcim/devices/` the column that indicates rack position has header `Position (U)` which is very long compared to the values on the column - they are usually just 1 or 2 digits.

I propose renaming this column header to simply `U`. It should save the precious horizontal screen real estate and allow at least 5-6 more characters for actual data.  
I would expect anyone using NetBox understands what having just "U" would mean, what do the other users think?

Please give your feedback and if there is some kind of reasonable consensus I would open a feature request issue.  
It could be simple enough change that I could open a PR to go with it but I did not look at what the change would involve exactly.

**Comment by jeremystretch** [comment]
Maybe "RU" to lend a tiny bit more context? My concern is primarily for potential translation efforts in the future.

**Reply by candlerb** [reply]
Maybe, although "RU" sounds a bit like device height to me. How about "Pos" ?  Changing "Rack Face" to just "Face" might be a good idea too.

Perhaps the position value shown should be in the form "U34", as it is in the drop-down menus, and the device detail page; that makes it clearer and more consistent.  However it might not sort properly.

**Reply by bluikko** [reply]
> Maybe, although "RU" sounds a bit like device height to me. How about "Pos" ? Changing "Rack Face" to just "Face" might be a good idea too.

`Rack Face`: have not thought about this since I don't have the space to keep it visible in the table. It indeed suffers from the same space-wasting syndrome with long heading vs short content.

`Pos` would meet the objective of being short, too. To me `U` is unambiguous and an obvious choice but let's see.

> 
> Perhaps the position value shown should be in the form "U34", as it is in the drop-down menus, and the device detail page; that makes it clearer and more consistent. However it might not sort properly.

I would always object to adding redundant information that takes horizontal space but I am very specific (hence opening this discussion).
